teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, parent_permission: true, name: 'Unknown')
+    super(id, age, parent_permission: parent_permission, name: name)
```

Comment:
- As I pointed out in the previous review, the Teacher is inheriting from the Person class, so each would have a unique id. There's no need to add the `id` in both parameters, kindly remove them.